### ISSUE TITLE
Considering add some helper `bracket` functions to Util in order to avoid begin* and end* syntax

### ISSUE THREAD

**Issue by futu2** [Anut-py/h-raylib#23]
```haskell
import Control.Monad.Catch (MonadMask, bracket_)
```

```haskell
drawing :: (MonadIO m, MonadMask m) => m () -> m ()
drawing = bracket_ (liftIO beginDrawing) (liftIO endDrawing)

mode3D :: (MonadIO m, MonadMask m) => RL.Camera3D -> m () -> m ()
mode3D c = bracket_ (liftIO $ beginMode3D c) (liftIO endMode3D)

mode2D :: (MonadIO m, MonadMask m) => RL.Camera2D -> m () -> m ()
mode2D c = bracket_ (liftIO $ beginMode2D c) (liftIO endMode2D)
```

**Comment by futu2** [comment]
similar things
```haskell
withWindow :: (MonadIO m, MonadMask m) => Int -> Int -> String -> Int -> (WindowResources -> m b) -> m b
withWindow w h title fps = bracket (liftIO $ initWindow w h title <* setTargetFPS fps) (liftIO . closeWindow)
```

**Comment by Anut-py** [comment]
Published in `4.6.0.5`
